README.md

```diff
@@ -26,28 +26,78 @@ Enter the provider directory and build the provider
 ```sh
 $ cd $GOPATH/src/github.com/PagerDuty/terraform-provider-pagerduty
 $ make build
-```
-
-## Using the provider
-
-Please refer to https://registry.terraform.io/providers/PagerDuty/pagerduty/latest/docs for
-examples on how to use the provider and detailed documentation about the
-Resources and Data Sources the provider has.
-
-## Developing the Provider
-
-If you wish to work on the provider, you'll first need [Go](http://www.golang.org) installed on your machine (version 1.11+ is *required*). You'll also need to correctly setup a [GOPATH](http://golang.org/doc/code.html#GOPATH), as well as adding `$GOPATH/bin` to your `$PATH`.
-
-To compile the provider, run `make build`. This will build the provider and put the provider binary in the `$GOPATH/bin` directory.
-
-```sh
-$ make build
 ...
 $ $GOPATH/bin/terraform-provider-pagerduty
 ...
 ```
 
-### Testing
+This will build the provider and put the provider binary in the `$GOPATH/bin` directory.
+
+## Usage
+
+Please refer to Terraform docs for [PagerDuty Provider](https://registry.terraform.io/providers/PagerDuty/pagerduty/latest/docs)
+for examples on how to use the provider and detailed documentation about the Resources and Data Sources the provider has.
+
+## Development
```

Comment:
Development has been renamed "Developing the Provider" and expanded into 4 sections:
* Setup Local Environment
* Setup Local PagerDuty Provider
* Run Dev Build with Local Terraform Module
* Setup Local Logs